internal/pkg/manifest/workload.go

```diff
@@ -78,6 +84,8 @@ var (
 	subnetPlacements = []string{string(PublicSubnetPlacement), string(PrivateSubnetPlacement)}
 
 	// Error definitions.
+	ErrAppRunnerInvalidPlatformWindows = errors.New("Windows is not supported for App Runner services")
```

Comment:
Shall we remove https://github.com/aws/copilot-cli/blob/9900bf423e4a13a2388a9bcd361991fd9329749d/internal/pkg/manifest/workload.go#L99

Reply:
App Runner is still not supported, unfortunately.

Reply:
Aren't they exactly the same except for the capital "E"... Also please stop working when OOO 🤖

Reply:
Oh yeah! Thanks for catching this!